hack/local-up-cluster.sh

```diff
@@ -88,6 +88,9 @@ ENABLE_POD_PRIORITY_PREEMPTION=${ENABLE_POD_PRIORITY_PREEMPTION:-""}
 # enable kubernetes dashboard
 ENABLE_CLUSTER_DASHBOARD=${KUBE_ENABLE_CLUSTER_DASHBOARD:-false}
 
+# enable dynamic audit
+ENABLE_APISERVER_DYNAMIC_AUDIT=${ENABLE_APISERVER_DYNAMIC_AUDIT:-true}
```

Comment:
Shouldn't this default to false? If we really wanted to default this value to true, then I feel like declaring `cluster/gce/util.sh` with the true value would probably be a little safer.

Reply:
Hey @logicalhan, sorry that was a debug measure, I intend on changing it before this moves out of WIP